content/posts/packit-030.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Packit 0.3.0"
+date: 2019-04-10
+draft: true
+---
+
+In [previous post](https://packit.dev/posts/packit-020/) we promised to release a new release every 2 weeks and we are already breaking that as it's been 3 weeks since then.
+
+Now the good news. [0.3.0](https://github.com/packit-service/packit/releases/tag/0.3.0), the new packit version, brings many new features and improvements.
+You can find a complete list in the [changelog](https://github.com/packit-service/packit/blob/master/CHANGELOG.md#030).
+<!--more-->
+
+## Features
+
+* Packit supports [Source-git](https://packit.dev/source-git/).
```

Comment:
I would leave that to 0.4.0 to be sure it works well. (I'll send some fixes and tests for that.)